cfme_perf.sh

```diff
@@ -7,12 +7,36 @@ else
   YUM=yum
 fi
 
-sudo $YUM install -y python-virtualenv gcc postgresql-devel libxml2-devel libxslt-devel zeromq3-devel libcurl-devel redhat-rpm-config  gcc-c++  freetype-devel libffi-dev
+sudo $YUM install -y python-virtualenv \
+                     gcc \
+                     postgresql-devel \
+                     libxml2-devel \
+                     libxslt-devel \
+                     libcurl-devel \
+                     redhat-rpm-config \
+                     gcc-c++ \
+                     python-virtualenv \
+                     libffi-devel \
+                     libpng \
+                     libpng-devel \
+                     freetype-devel \
+                     freetype-devel \
+                     openssl-devel
+
+echo "### Creating Virtual Environment: .cfme_perf"
```

Comment:
We should add .cfme_perf to the .gitignore for one less item to appear in git status and prevent virtualenv files from sneaking into a commit.